qemu/tests/cfg/multi_nics_stress.cfg

```diff
@@ -24,3 +22,6 @@
         server_path_win = "c:\\"
         client_path_win = "c:\\"
         test_protocols = "TCP_STREAM TCP_MAERTS UDP_STREAM TCP_RR TCP_CRR UDP_RR"
+    Linux:
+        pre_cmd = "for nic in `ls /sys/class/net|grep -v lo`;do arp -a|grep -v $nic && dhclient $nic;done"
```

Comment:
pre_cmd is ok for me.  but if adding "-v" param for dhclient cmd, it will give more detailed information. I think this is useful during debugging if the case fail.

pre_cmd = "for nic in `ls /sys/class/net|grep -v lo`;do arp -a|grep -v $nic && dhclient -v $nic;done"